File: 168-Excel表列名称.py
class Solution:
    def convertToTitle(self, n: int) -> str:
        mystack = []
        while n != 0:
            a = n % 26
            if a == 0:
                mystack.append(26)
                n = n // 26 - 1
            else:
                mystack.append(a)
                n = n // 26

        s = ''
        while mystack:
            s += chr(mystack.pop() + 64)

        return s

File: 168-Excel表列名称.cpp
class Solution {
public:
    string convertToTitle(int n) {
        stack<int> mystack;
        int a;
        while(n != 0){
            a = n % 26;
            if(a == 0){
                mystack.push(26);
                n = n / 26 - 1;
            }
            else{
                mystack.push(a);
                n = n / 26;
            }
        }

        string s;
        while(!mystack.empty()){
            s += char(mystack.top() + 64);
            mystack.pop();
        }

        return s;
    }
};
